src/azure-cli/setup.py

```diff
@@ -65,7 +65,7 @@
     'azure-mgmt-applicationinsights~=0.1.1',
     'azure-mgmt-appconfiguration~=0.4.0',
     'azure-mgmt-authorization~=0.52.0',
-    'azure-mgmt-batch~=7.0',
+    'azure-mgmt-batch~=8.0',
```

Comment:
Currently, we prefer locking this version to `~=8.0.0`. Please correct me if I miss something. @fengzhou-msft @haroldrandom